src/main/scala/firrtl/annotations/JsonProtocol.scala

```diff
@@ -22,43 +25,70 @@ trait HasSerializationHints {
 /** Wrapper [[Annotation]] for Annotations that cannot be serialized */
 case class UnserializeableAnnotation(error: String, content: String) extends NoTargetAnnotation
 
-object JsonProtocol {
+object JsonProtocol extends LazyLogging {
+  val GetClassPattern = "[^']*'([^']+)'.*".r
+
   class TransformClassSerializer
       extends CustomSerializer[Class[_ <: Transform]](format =>
         (
-          { case JString(s) => Class.forName(s).asInstanceOf[Class[_ <: Transform]] },
-          { case x: Class[_] => JString(x.getName) }
+          {
+            case JString(s) => Class.forName(s).asInstanceOf[Class[_ <: Transform]]
+          },
+          {
+            case x: Class[_] => JString(x.getName)
+          }
```

Comment:
This change just seems to be adding [unnecessary] whitespace? Why?

Reply:
Same for many below, seems to make it verbose for not any real readability benefit. It also distracts from the PR with lots of unrelated diffs.